Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -34,6 +34,29 @@ Sealed Resources version=2 rules=12 files=1410
 Internal requirements count=1 size=176
 ```
 
+### Determining Whether the App is Debuggable
+
+#### Overview
+
+Debugging iOS applications can be done using Xcode, which embeds a powerful debugger called lldb. Lldb is the default debugger since Xcode5 where it replaced GNU tools like gdb and is fully integrated in the development environment. While debugging is a useful feature when developing an app, it has to be turned off before releasing apps to stores.
```

Comment:
Shouldn't we be promoting just one store, so "to the App store" ?

Reply:
it would be better to say to the app-store or to your enterprise program.